### PR TITLE
WS-977: Re-add masters programs to algolia index

### PR DESCRIPTION
**Pull request by reillz10**

### CODE REVIEW

File: course_discovery/apps/course_metadata/algolia_proxy_models.py [hunk 1]
```diff
@@ -288,6 +288,10 @@ def program_types(self):
     @property
     def availability_level(self):
         all_courses = self.courses.all()
+        
+        print('rickie rulez ' + self.type.slug)
```
Comment (rgraber) [inline course_discovery/apps/course_metadata/algolia_proxy_models.py]: tempting but should probably remove. also can we comment about why we're doing this

Comment (reillz10) [reply course_discovery/apps/course_metadata/algolia_proxy_models.py]: omg great catch..... I'm going to --amend push so no one knows i did this...

**Review by rgraber — APPROVED** [review]
LGTM

**Review by mikix — APPROVED** [review]

File: course_discovery/apps/course_metadata/algolia_proxy_models.py [hunk 2]
```diff
@@ -289,6 +289,12 @@ def program_types(self):
     def availability_level(self):
         all_courses = self.courses.all()
 
+        # Master's programs don't have courses in the same way that our other programs do.
```
Comment (albemarle) [inline course_discovery/apps/course_metadata/algolia_proxy_models.py:292]: 👍 for comment.

**Pull request merged**